samples/python/kalman.py

```diff
@@ -86,7 +86,7 @@ def draw_cross(center, color, d):
 
             cv2.imshow("Kalman", img)
 
-            code = cv2.waitKey(100)
+            code = cv2.waitKey(100) & 0xFF
```

Comment:
For python a question, can we simply add CV_EXPORTS_W to static inline functions? Then I can cover it in python also!